web/src/inbox_ui.js

```diff
@@ -250,6 +250,7 @@ function format_stream(stream_id) {
         // Will be displayed if any topic is visible.
```

Comment:
<img width="868" alt="Screenshot 2023-09-19 at 6 14 54 PM" src="https://github.com/zulip/zulip/assets/25124304/107fd762-b4bc-4534-9a22-baa7bf805f84">

Should I add mention indicators for DMs? I am not it is required.

Reply:
Regarding mention indicators in DMs, We should match the left sidebar behavior; I don't remember what we do there.

I think the mention indicators are too far away from the unread counts here; is that the same spacing we have in recent conversations?

Reply:
Added a commit in #26822 to reduce space.